lambda/moviebotFunction.js

```diff
@@ -261,13 +130,22 @@ function findMovie(intentRequest, callback) {
         callback(confirmIntent(sessionAttributes, 'ContinueFinding', {}, msg, movieToResponseCards(singleMovieList)))
     }).catch((err) => {
         console.log(err)
-        //error object structure
         if (err.type === "Validation") {
-
+            if (intentName === "FindMovieByActor") {
```

Comment:
Refer again to https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Error